LEGO1/mxrect32.h

```diff
@@ -1,6 +1,9 @@
 #ifndef MXRECT32_H
 #define MXRECT32_H
 
+#define MxRect32GetWidth(rect) (rect.m_right - rect.m_left) + 1
+#define MxRect32GetHeight(rect) (rect.m_bottom - rect.m_top) + 1
+
```

Comment:
Does it affect the match percentage if these are inline getter methods instead?

Reply:
It shouldn't, but that's how microsoft does the rectangle width/height macros, so I assumed that's how mindscape did

Reply:
Nope, it doesn't affect the percentage, but it's cleaner.